tensorboard/webapp/metrics/views/card_renderer/image_card_test.ts

```diff
@@ -757,5 +757,83 @@ describe('image card', () => {
       slider = fixture.debugElement.query(By.css('mat-slider'));
       expect(slider.nativeElement.getAttribute('aria-valuenow')).toBe('2');
     });
+
+    describe('range selection', () => {
```

Comment:
Can you add a test for when the entire range is outside the range of available images? 
Ex:
Start step = 50
End step = 60

Reply:
it's covered by `'dose not moves slider thumb when selected time is clipped',` I set start step 55 and end step 65